journal/modernized/33-chapter-30.adoc

```diff
@@ -243,7 +243,7 @@ where we partook of a repast, which had been kindly prepared for our accommodati
 The king or principal chief Makea, joined us;
 he was desirous of making a present of supplies for the vessel;
 this however was declined, and his kind intention acknowledged with gratitude.
-This chiefs sister, once dined on board the Henry Freeling,
+This chief`'s sister, once dined on board the Henry Freeling,
```

Comment:
Not sure if I did the apostrophe here right in both editions. I didn't wanna waste time looking for it in asciidoc somewhere else, and I just remembered it's probably in the style guide. :)